docs/administration/workspaces.md

```diff
@@ -0,0 +1,12 @@
+# Multiple Workspaces
+
+Each Workspace in Eppo is connected to one data warehouse and has it's own Facts, Metrics, Experiments, and Flags. By default each customer has one Workspace.
```

Comment:
Maybe would be nice to have one or two sentences to start the doc that explains the motivation for docs